### PR TITLE
Fix import ignores newly-added files

### PR DESCRIPTION
**Pull request by a-schuetz**
Newly added files are imported to server side source control instead of ignored.

### CODE REVIEW

File: cls/SourceControl/Git/Utils.cls [hunk 1]
```diff
@@ -1138,10 +1138,12 @@ ClassMethod ImportRoutines(force As %Boolean = 0) As %Status
     for  {
         set internalName = $order(itemList(internalName))
         quit:internalName=""
-        if '..IsInSourceControl(internalName) {
-            continue
+        #dim sc as %Status
+        if ..IsInSourceControl(internalName) {
+            set sc = ..ImportItem(internalName, force)            
+        } else {
+            set sc = ..AddToServerSideSourceControl(internalName)
```
Comment (isc-tleavitt) [inline cls/SourceControl/Git/Utils.cls:1145]: Are such newly added files actually imported? Looking at the diff I'd think not, but maybe this happens deeper down - haven't actually tested. Presumably we'd always want to call ImportItem?

**Review by isc-tleavitt — COMMENTED** [review]
One note here re: testing.

Note, I have a fix in progress that covers some semi-related issues too. (Should have explicitly stated that.) Feel free to take a look at the import-export-fixes branch.

**Comment by a-schuetz** [comment]
fixes #217

File: cls/SourceControl/Git/Utils.cls (hunk 1 above)
Comment (a-schuetz) [reply cls/SourceControl/Git/Utils.cls:1145]: Yes, `AddToServerSideSourceControl` first updates the global and than calls `ImportItem`.

```Objectscript
ClassMethod AddToServerSideSourceControl(InternalName As %String) As %Status
{
     #dim i as %Integer
    #dim ec as %Status = $$$OK
    for i = 1:1:$length(InternalName, ",") {
        #dim item as %String = ..NormalizeExtension($piece(InternalName, ",", i))
        set @..#Storage@("items", item) = ""
        #dim sc as %Status =  ..ImportItem(item, 1)
        if 'sc {
            set ec = $$$ADDSC(ec, sc)
        }
    }
    quit ec
}
```

**Comment by isc-tleavitt** [comment]
@a-schuetz I've folded your change into a larger overhaul of import/export that fixes a bunch of other issues too. See #220. As such, I'm closing this PR, but thank you for your contribution and for reporting the issue in the first place!

**Pull request closed without merging**